src/Stratis.Bitcoin.IntegrationTests/Builders/NodeGroupBuilder.cs

```diff
@@ -33,6 +32,24 @@ public NodeGroupBuilder StratisPowNode(string nodeName)
             return this;
         }
 
+        public NodeGroupBuilder CreateStratisPowMiningNode(string nodeName)
```

Comment:
Shall we update the bitcoin pow method on line 29 from `StratisPowNode(` to `CreateStratisBitcoinPowNode(` or maybe even `CreateBitcoinPowNode(` ?

Reply:
We will get to renaming these runners properly soon!

Reply:
ok